tests/std/tests/VSO_0971246_legacy_await_headers/env.lst

```diff
@@ -3,8 +3,8 @@
 
 RUNALL_INCLUDE ..\prefix.lst
 RUNALL_CROSSLIST
-PM_CL="/EHsc /MT /std:c++latest"
-PM_CL="/EHsc /MT /await /std:c++latest"
+PM_CL="/EHsc /MT /std:c++latest /permissive-"
+PM_CL="/EHsc /MT /await /std:c++latest /permissive-"

```

Comment:
Previously, we had no strict coverage for C1XX here. Now, we have no lax coverage.